packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx

```diff
@@ -360,7 +360,7 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
             </Stack>
           </CallAdapterProvider>
         </Stack.Item>
-        {!props.mobileView && (
+        {!props.mobileView && (isEnabled(options.peopleButton) || isEnabled(options.chatButton)) && (
```

Comment:
we support injecting buttons here, what if these buttons are not enabled but a custom button is injected?

Reply:
good point, will account for that in the logic